### ISSUE TITLE
Plans to switch from forked libusb to upstream libusb?

### ISSUE THREAD

**Issue by tony--** [tessel/node-usb#161]
Are there any plans to switch?
Is the lack of hotplug support for windows the primary blocker?
When [hotplug support for windows is added to libusb](https://github.com/libusb/libusb/issues/86) will this module switch to using libusb instead of the fork?

**Comment by kevinmehall** [comment]
Yes, hotplug support is the only difference on the fork, and it would be great to switch to upstream libusb.
